aspnetcore/signalr/javascript-client.md

```diff
@@ -98,7 +98,140 @@ Use the [configureLogging](/javascript/api/%40aspnet/signalr/hubconnectionbuilde
 
 ## Reconnect clients
 
-The JavaScript client for SignalR doesn't automatically reconnect. You must write code that will reconnect your client manually. The following code demonstrates a typical reconnection approach:
+::: moniker range=">= aspnetcore-3.0"
+
+### Automatically reconnect
+
+The JavaScript client for SignalR doesn't automatically reconnect by default. Automatic reconnects can be enabled by calling the [withAutomaticReconnect](/javascript/api/%40aspnet/signalr/hubconnectionbuilder#withautomaticreconnect) method on [HubConnectionBuilder](/javascript/api/%40aspnet/signalr/hubconnectionbuilder):
+
+```javascript
+const connection = new signalR.HubConnectionBuilder()
+    .withUrl("/chatHub")
+    .withAutomaticReconnect()
+    .build();
+```
+
+Without any parameters, `withAutomaticReconnect()` configures to client to wait 0, 2, 10 and 30 seconds respectively before trying up to 4 reconnect attempts.
```

Comment:
I don't like the "between each reconnect attempt" phrasing because the first delay take place *before* the first reconnect attempt. That's why I was careful to the delay is before, not between, each reconnect attempt.